### PR TITLE
[IMP] Odoo: Add image description

### PR DESCRIPTION
**Pull request by Goaman**

### CODE REVIEW

File: packages/plugin-odoo/src/Odoo.ts [hunk 1]
```diff
@@ -279,6 +279,18 @@ export class Odoo<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
                     return [button];
                 },
             },
+            {
+                id: 'OdooDescriptionActionable',
+                async render(): Promise<ActionableNode[]> {
+                    const button = new ActionableNode({
+                        name: 'describe-image',
+                        label: 'Description',
+                        commandId: 'describeImage',
```
Comment (dmo-odoo) [inline packages/plugin-odoo/src/Odoo.ts:288]: Wait, I can't find any occurence of `describeImage` in current master, and it is not implemented by this branch so... where is it ?

Comment (Goaman) [reply packages/plugin-odoo/src/Odoo.ts:288]: In our Odoo master branch.

Comment (dmo-odoo) [reply packages/plugin-odoo/src/Odoo.ts:288]: Wait, so this plugin adds a button that calls a command that is neither in the plugin itself nor in one of its dependencies ?

Comment (Goaman) [reply packages/plugin-odoo/src/Odoo.ts:288]: Well, it's already the case for all the command that need to run Odoo code. See these commands in wysiwyg.js in Odoo:
```

            customCommands: Object.assign({
                openMedia: {handler: this.openMediaDialog.bind(this)},
                openTextColorPicker: {handler: this.toggleTextColorPicker.bind(this)},
                openBackgroundColorPicker: {handler: this.toggleBackgroundColorPicker.bind(this)},
                openLinkDialog: {handler: this.openLinkDialog.bind(this)},
                discardOdoo: {handler: this.discardEditions.bind(this)},
                saveOdoo: {handler: this.saveToServer.bind(this)},
                cropImage: {handler: this.cropImage.bind(this)},
                transformImage: {handler: this.transformImage.bind(this)},
                describeImage: {handler: this.describeImage.bind(this)},
            }, this.options.customCommands),
```

**Pull request merged**